dashboard/app/public_json_api.go

```diff
@@ -8,15 +8,16 @@ import "encoding/json"
 // publicApiBugDescription is used to serve the /bug HTTP requests
 // and provide JSON description of the BUG. Backward compatible.
 type publicAPIBugDescription struct {
-	Version    int         `json:"version"`
-	Title      string      `json:"title,omitempty"`
-	FixCommits []fixCommit `json:"fix-commits,omitempty"`
+	Version     int         `json:"version"`
+	Title       string      `json:"title,omitempty"`
+	FixCommits  []gitCommit `json:"fix-commits,omitempty"`
+	CauseCommit *gitCommit  `json:"cause-commit,omitempty"`
 	// links to the discussions
 	Discussions []string                    `json:"discussions,omitempty"`
 	Crashes     []publicAPICrashDescription `json:"crashes,omitempty"`
 }
 
-type fixCommit struct {
+type gitCommit struct {
```

Comment:
Consider calling this commit. The bisection code itself is VCS agnostic, so this might not be a git commit on other OSs.

Reply:
done